### ISSUE TITLE
missing core/once to libraries.yml

### ISSUE THREAD

**Issue by john4190** [openeuropa/oe_theme#1313]
in order to fully support js asset injection i strongly suggest to include core/once in theme dependencies

**Comment by brummbar** [comment]
Js assets should be libraries, and they can declare their dependencies.

**Comment by brummbar** [comment]
... unless you mean in the current libraries we have. On that I agree :)

**Comment by john4190** [comment]
yes but i am not trying to create a new module, i am trying to exploit ECL
components in order to make search page as compliant as possible to
European union guidelines. Moreover fortunately there are modules for
injecting small pieces of javascript without creating a new module and
without disrupting everything

Il giorno mer 6 set 2023 alle ore 15:53 Francesco ***@***.***>
ha scritto:

**Comment by john4190** [comment]
anyway yes having core/once helps you as well if you have to handle events in the drupal way and keep everything compatible with drupal 10
